### DISCUSSION TITLE
How to silence Mason's LSP installation failure messages?

### DISCUSSION THREAD

**Discussion by njvack** [LazyVim/LazyVim#4648] [Q&A]
Hey there,

I'm trying out lazyvim — most notably, I'm trying out using it on _all_ the various unix machines I use (I think I have 7-10?) so I have something like a vaguely-consistent text editing experience. So I'm syncing my .config/nvim directory between machines rather than starting from scratch everywhere.

On some of these computers, I've been able to make sure all the LSP stuff is... well, at least in a state where it doesn't raise errors when I start it. On others, I'm never going to install rubocop or whatever because I don't need it in that context. But every time I start nvim to edit a shell script, the UI is peppered with installation messages and then warnings about failures to install stuff.

Can I... make this stop? I'd be okay with a variety of solutions: Having Mason install stuff (I think this is a correct way to explain what's happening?) only on request and not automatically, muting installation and error messages until I ask for them, or ???

Apologies if I'm explaining any of this badly; I'm quite new to using vi as anything but the most bare-bones editor available and there is quite a lot of magic happening here.

Many thanks!

**Comment by GerritWanderer** [comment]
@njvack I guess most of the LSP specific configs are activated through the `lazyvim.json` file in the root directory of the LayzVim directory?
You can create multiple json files (eg. `lazyvim-with_lsp.json` and `lazyvim-without_lsp.json`) and after syncing your dotfiles to your machines, just create a symlink with the config you want to activate on your current machine.
I do something similar for my zsh config files depending on the OS of the machine (MacOS or Linux)
```
ln -s .zshrc.macbook.zsh .zshrc.zsh
```
